### ISSUE TITLE
Windows Service - nssm last stable version from 2014

### ISSUE THREAD

**Issue by tinohager** [minio/minio-service#25]
I think it makes sense to switch to an other solution. I have found this one [winsw](https://github.com/kohsuke/winsw)

- Download [WinSW.NET4.exe](https://github.com/kohsuke/winsw/releases/download/winsw-v2.2.0/WinSW.NET4.exe)
- Rename the `WinSW.NET4.exe` to `minio-service.exe`
- Create a xml file `minio-service.xml` insert the configuration below
- Open a `cmd` as Administrator and execute `minio-service.exe install`

```xml
<service>
  <id>MinIO</id>
  <name>MinIO</name>
  <description>MinIO is a high performance object storage server</description>
  <executable>minio.exe</executable>
  <arguments>server C:\Temp\minio</arguments>
  <logmode>rotate</logmode>
</service>
```

**Comment by kannappanr** [comment]
@tinohager Thanks for the suggestion. We will check this out

**Comment by tinohager** [comment]
Powershell Script for easy installation
[install-service.zip](https://github.com/minio/minio-service/files/3349178/install-service.zip)

**Comment by tinohager** [comment]
Fix TLS error on download
[install-service.zip](https://github.com/minio/minio-service/files/3349205/install-service.zip)

**Comment by harshavardhana** [comment]
Feel free to send a PR @tinohager

**Comment by tinohager** [comment]
https://github.com/minio/minio-service/pull/26

**Comment by harshavardhana** [comment]
Fixed by #26
